lib/generators/rodauth/templates/app/views/rodauth/tailwind/otp_setup.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :title, rodauth.otp_setup_page_title %>
+<%= form_with url: rodauth.otp_setup_path, method: :post, data: { turbo: false } do |form| %>
+<%= form.hidden_field rodauth.otp_setup_param, value: rodauth.otp_user_key, id: "otp-key" %>
+<%= form.hidden_field rodauth.otp_setup_raw_param, value: rodauth.otp_key, id: "otp-hmac-secret" if rodauth.otp_keys_use_hmac? %>
+<div class="flex items-center min-h-screen dark:bg-gray-900 bg-gray-50 bg-white rounded-lg shadow-xl">
+	<div class="flex-1 h-full max-w-4xl mx-auto">
+		<div class="flex flex-col md:flex-row">
+			<div class="flex md:hidden">
+				<%== rodauth.otp_qr_code %>				
+			</div>
+			<div class="h-32 md:h-auto md:w-1/2 hidden md:flex">
+				<div class="object-cover w-full h-full">
+					<%== rodauth.otp_qr_code %>					
+				</div>
+			</div>
+			<div class="flex items-center justify-center p-6 sm:p-12 md:w-1/2">
+				<div class="w-full">
+					<div>
+						<p class="dark:text-gray-100 ">
+							<b><%= rodauth.otp_secret_label %>: </b>
```

Comment:
MDN [advises against](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) using the `<b>` tag, and instead recommends using `<strong>` or styling with CSS.

Reply:
Fixed with: [2a2f6ba](https://github.com/janko/rodauth-rails/pull/114/commits/2a2f6ba8edf0b16a3e2842dc793725f8f8e47c0b)